server/DAL/Configuration/JudgeRoleConfiguration.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Core.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace DAL.Configuration;
+
+public class JudgeRoleConfiguration
+    : IEntityTypeConfiguration<JudgeRole>
+{
+    public void Configure(EntityTypeBuilder<JudgeRole> builder)
+    {
+        builder.ToTable("judge_roles");
+
+        builder.HasKey(jr => jr.RoleName);
+        
+        builder
+            .Property(jr => jr.RoleName)
+            .HasColumnName("role_name")
+            .HasColumnType("varchar(50)")
+            .IsRequired();
+
+        builder.HasData(
+            new JudgeRole() 
+            {
+                RoleName = "side_judge"
+            },
+            new JudgeRole() 
+            {
+                RoleName = "referee"
+            },
+            new JudgeRole() 
+            {
+                RoleName = "chief_judge"
+            },
+            new JudgeRole() 
+            {
+                RoleName = "deputy_chief_judge"
+            },
+            new JudgeRole() 
+            {
+                RoleName = "reserve_judge"
+            }
+        );
```

Comment:
I suggest moving data seeding to separate classes, so that it's easier to separate it from the table configuration. Also there could be a kind of check to see if a table already has data. Something like `if (!table.HasData()) table.Seed()`. But it's unnecessary as long as we don't have much data to seed I think

Reply:
Good point. I have plans to add more data seeding later on, then I will make sure to do everything properly. For now I think it can stay like this.